styles.css

```diff
@@ -0,0 +1,541 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+:root {
+  --Primary: #172b4d;
+  --Secondary: #36b37f;
+}
+
+.fa-bars {
+  color: #28352f;
+}
+
+.burger {
+  display: none;
+}
+
+.flex-all {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.flex-center {
+  text-align: center;
+  display: flex;
+  align-items: center;
+}
+
+.flex-new {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.grid {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+}
+
+.widh-marg {
+  max-width: 1020px;
+  margin: auto;
+}
+
+.mb {
+  margin-bottom: 3rem;
+}
+
+.mt {
+  margin-top: 7rem;
+}
+
+* {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.hero {
+  height: 100vh;
+  background: url(assest/img/bgimg/Header-llustration-desktop.png);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.header-content {
+  width: 95%;
+  margin: auto;
+  padding-top: 1.5%;
+  position: fixed;
+}
+
+.logo {
+  visibility: hidden;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  width: 57%;
+  gap: 35px;
+}
+
+.nav-list a {
+  color: var(--Primary);
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.nav-list li a:hover {
+  color: var(--Secondary);
+}
+
+.contant {
+  flex-direction: column;
+  gap: 24px;
+  justify-content: center;
+  height: 100vh;
+}
+
+h1 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Primary);
+}
+
+h2 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Secondary);
+}
+
+p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: var(--Primary);
+  flex: none;
+  flex-grow: 0;
+  max-width: 750px;
+}
+
+.social {
+  gap: 20px;
+  margin-top: 3%;
+}
+
+.next {
+  margin-top: 5%;
+  display: flex;
+}
+
+.head {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--Primary);
+}
+
+.img-multi {
+  background-image: url(assest/img/recent/holder.png);
+  background-size: cover;
+  height: 260px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  flex-direction: column;
+  border: 1px solid #ebf0ee;
+  border-radius: 10px;
+}
+
+.text {
+  margin-bottom: 25px;
+  flex-direction: column;
+}
+
+h3 {
+  margin-bottom: 5%;
+}
```

Comment:
- [ ] For the individual cards, kindly adopt the `font-size`, `font-weight` and `color` used for the title of the cards. The `font-size` it is `24px,` the `color` is `#3a4a42` and the `font-weight` is `600`

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219487286-1cc43ccd-2c81-4a5a-b432-03401fd9759d.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/219487366-f067b3fd-9701-47bb-843b-f187972960a4.png)